### ISSUE TITLE
要望：(Windows的な意味での)複数ウィンドウの対応

### ISSUE THREAD

**Issue by yasuhirokimura** [kmiya-culti/RLogin#45]
複数ウィンドウなら既に対応していると言われそうですが、ここではRLogin的な意味でのウィンドウではなく、Windows的な意味でのウィンドウです。Webブラウザの、ディスプレイ上に複数のウィンドウが表示されて、更にその各々のウィンドウに複数のタブが表示されて、というような状態をイメージしていただけると、私の要望が理解していただけると思います。一台のPCに複数のディスプレイを接続しているので、それぞれのディスプレイにRLoginのウィンドウを表示させて使いたい、というのが要望の背景です。そもそも対応可能なのかも判りませんし、対応可能だとしてもプログラムのかなり大規模な変更になると思うので、作者様のToDoリストの最後にでも追加しておいていただいて、余裕があるときにでも対応を検討していただければありがたいです。

**Comment by ttdoda** [comment]
よくわからないのですが、RLogin を複数起動するのでは駄目なのでしょうか?
何か違う動作を期待しているのでしょうか?

**Comment by yasuhirokimura** [comment]
@ttdoda 例えばブラウザだと、あるウィンドウに属するタブを別のウィンドウに移動させることができますが、アプリケーションの複数起動だと多分そういうことはできないと思います。なので、そういう点も含めて、複数ウィンドウに対応していただけるとありがたいなあ、と。

**Comment by belgianbeer** [comment]
@yasuhirokimura お望みの機能はすでにありますよ。Rlogin.exe を起動した状態でタスクバーのRlogin.exe のiconを右クリックするとコンテキストメニューに「Rlogin MFCアプリケーション」というのがあるので、それを選べば別ウィンドウでRlogin.exe が起動します。

これをメニューからできるとキーボードで操作できるのですが、現状は無いですよね？

あと個人的にはRlogin.exe のWindowという表現は紛らわしいと思うのですが、「タブ」とも微妙に違うのでなんとも。

**Comment by yasuhirokimura** [comment]
@belgianbeer いや、それは単にRloginを複数起動しているだけです。それをやった後にタスクマネージャーを起動すると、「Rlogin MFCアプリケーション」が複数動作しているはずです。
